client/lib/feed-stream-store/feed-stream.js

```diff
@@ -518,6 +524,18 @@ export default class FeedStream {
 				to: this.pendingDateAfter,
 			} );
 		}
+		forEach( this.pendingPostKeys, postKey => {
+			// conversations!
+			if ( postKey.comments ) {
+				const post = FeedPostStore.get( postKey );
+				reduxDispatch( {
+					type: COMMENTS_RECEIVE,
+					siteId: post.site_ID,
+					postId: post.ID,
```

Comment:
site id and post id should already be on the post key. Here they would be `site_ID` and `ID`.

Reply:
the postKey has them as `blogId` and `postId`

Reply:
ah, right, this is past the api response. so wish we had types. 

either way, i don't think you _need_ the post store here. you already have all the info you need.